less/circle-background.less

```diff
@@ -0,0 +1,164 @@
+@circle-bg-xs-diameter: 25%;
+@circle-bg-sm-diameter: 50%;
+@circle-bg-md-diameter: 120%;
+@circle-bg-lg-diameter: 200%;
+@circle-bg-xl-diameter: 400%;
+@circle-bg-translate-to-edge: 5%;
+@circle-bg-translate-to-center:50%;
+@circle-bg-translate-to-center-asymmetric:35%;
+
+.circle-bg {
+  position: relative;
+  z-index: 1;
```

Comment:
That z-index is necessary when the circle background is used inside an element that has already a background. What king of issue could cause?